framework/Models/PostProcessors/Validations/PPDSS.py

```diff
@@ -0,0 +1,412 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Created on January XX, 2021
+
+@author: yoshrk
+"""
+#External Modules------------------------------------------------------------------------------------
+import numpy as np
+from scipy.interpolate import interp1d
+from scipy.integrate import simps
+import xarray as xr
+import os
+from collections import OrderedDict
+import itertools
+import copy
+#External Modules End--------------------------------------------------------------------------------
+
+#Internal Modules------------------------------------------------------------------------------------
+from utils import utils
+from utils import InputData, InputTypes
+from utils import xmlUtils
+import Files
+import DataObjects
+from ..ValidationBase import ValidationBase
+#Internal Modules End--------------------------------------------------------------------------------
+
+class PPDSS(ValidationBase):
+  """
+    DSS Scaling class.
+  """
+  @classmethod
+  def getInputSpecification(cls):
+    """
+      Method to get a reference to a class that specifies the input data for
+      class cls.
+      @ In, cls, the class for which we are retrieving the specification
+      @ Out, inputSpecification, InputData.ParameterInput, class to use for
+        specifying input of cls.
+    """
+    inputSpecification = super(PPDSS, cls).getInputSpecification()
+    # Have added the new pivotParameters for feature and target. The original has been commented out.
+    pivotParameterFeatureInput = InputData.parameterInputFactory("pivotParameterFeature", contentType=InputTypes.StringType,
+                                                                descr="""Pivot parameter for feature inputs""")
+    inputSpecification.addSub(pivotParameterFeatureInput)
+    pivotParameterTargetInput = InputData.parameterInputFactory("pivotParameterTarget", contentType=InputTypes.StringType,
+                                                                descr="""Pivot parameter for target inputs""")
+    inputSpecification.addSub(pivotParameterTargetInput)
+    scaleTypeInput = InputData.parameterInputFactory("scale", contentType=InputTypes.StringType,
```

Comment:
Please try to use ```InputTypes.makeEnumType``` to define a new type for scale parameter, see the following for an example:
![image](https://user-images.githubusercontent.com/7321071/142457852-fd3cce6f-8355-405c-b6d9-71faa02a3eea.png)